app/views/order_mailer/order.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+</head>
+<body>
+<h1> Customer: <%= @customer_email  %>, Pickup time: <%= @pickup_time  %></h1>
+<% @cart.each do |item| %>
+  <h2><%=  item[:product_name] %> <%= item[:ingredient_name] unless item[:ingredient_name].nil? %> x<%= item[:item_quantity] %></h2>
+<% end %>
+</body>
+</html>
```

Comment:
Good for first iteration

Reply:
Can we add product_id as an order number into mailer?

Reply:
If we did that then the people with same items in their order would have same order number